src/aeromancy/wandb_tracker.py

```diff
@@ -47,6 +50,9 @@ def __init__(
         # For "https://github.com/x/y.git", we'd pull out "y" as the job name
         job_name = runtime_environment.git_repo_name
 
+        tags = self.tags or []
+        tags.insert(f"git/{runtime_environment.git_branch}")
```

Comment:
`add` 

(`insert` would fail if the user passed a set- can you test that easily?)

Reply:
Done. Sadly, I don't think we can test that easily -- I think we'd need to do some dependency injection to put a mock wandb module in?

As it stands, it will create a `TypeError`, which seems okay? Incidentally, just found several more instances of `tags`-as-`list` in `action.py` and `action_runner.py` which I converted. Manually tested that tags still propagate from `--tags` command-line flag to W&B web UI.